R/page.R

```diff
@@ -149,7 +149,8 @@ page_sidebar <- function(..., sidebar = NULL, title = NULL, fillable = TRUE, fil
 #'   host URL of the page is displayed by default).
 #' @export
 page_navbar <- function(..., title = NULL, id = NULL, selected = NULL,
-                        sidebar = NULL, fillable = FALSE, fill_mobile = FALSE,
+                        sidebar = NULL, fillable = TRUE, fill_mobile = FALSE,
+                        fill_gap = NULL, fill_padding = NULL,
```

Comment:
Maybe it's better for it to be just `gap` and `padding` since that's more consistent with the other APIs